Parser/tokenizer.c

```diff
@@ -1460,15 +1458,15 @@ tok_get(register struct tok_state *tok, char **p_start, char **p_end)
             if (c == 'j' || c == 'J')
                 goto imaginary;
 #endif
-            if (c != 'o' || c != 'O') {
-                char buf[100];
-                if (Py_Py3kWarningFlag) {
-                    if (PyErr_WarnExplicit_WithFix(PyExc_Py3xWarning, "using just a '0' prefix for octal literals is not supported in 3.x", 
-                                                   "use the '0o' prefix for octal integers, if you intended the integer to be decimal", tok->filename, tok->lineno, NULL, NULL)) {
-                        return NULL;
-                    }
-                }
-            }
+            // if (c != 'o' || c != 'O') {
```

Comment:
I wasn't expecting this to be commented out, but maybe I've misunderstood how it relates to string warnings?